### ISSUE TITLE
[need help] connect to calibre OPDS and wireless

### ISSUE THREAD

**Issue by cramoisi** [koreader/koreader#3954]
* KOReader version: 1711
* Device: KA1

I don't know if it's a bug or if it's me (probably the lastest) so let's again make i'm six.

Here is my config : 
a routeur with wifi (192.168.1) and 2 computers, 1 reader, which ip's are fixed by MAC adress : 192.168.0.2 (where calibre is running), 192.168.0.3 (reader), 192.168.0.4 (another pc) ; the rest is dhcped.

So when I launch calibre on 192.168.0.2, auth disabled, port 8080, listening on 0.0.0.0 (all) ;  the reader is unable to reach it (I see nothing in the log).

I got the same problem with calibre wireless connection (nothing found, no log entry).

But, I can connect with another computer web browser to the calibre library without problem from my lan or from internet.

Any idea ?

**Comment by NiLuJe** [comment]
You're missing a bit of router IP (which I'm going to assume is actually `192.168.0.1` given the context ;)).

What does the Kobo's browser say when trying to reach the Calibre library? Or, hell, just a ping to the box?

**Comment by cramoisi** [comment]
@NiLuJe  ok, it does nothing. stay forever on the homepage , that's all. google and wikipedia works and I can reach http://www.perdu.com which is the first website i went, back in 1997 :)

**Comment by cramoisi** [comment]
@dengste : I don't have a proxy configured. For all the other questions, text below : 

## cd /Volumes/KOBOeReader/.adds/misc ##
Last login: Wed May 16 19:30:53 on ttys001
Restored session: Mer 16 mai 2018 19:31:45 CEST
MacBook:~ sebastien$ ping 192.168.0.4
PING 192.168.0.4 (192.168.0.4): 56 data bytes
64 bytes from 192.168.0.4: icmp_seq=0 ttl=64 time=124.167 ms
64 bytes from 192.168.0.4: icmp_seq=1 ttl=64 time=24.032 ms
64 bytes from 192.168.0.4: icmp_seq=2 ttl=64 time=68.115 ms
64 bytes from 192.168.0.4: icmp_seq=3 ttl=64 time=5.717 ms
64 bytes from 192.168.0.4: icmp_seq=4 ttl=64 time=100.096 ms
^C
--- 192.168.0.4 ping statistics ---
5 packets transmitted, 5 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 5.717/64.425/124.167/44.572 ms
MacBook:~ sebastien$ 
MacBook:~ sebastien$ ssh root@192.168.0.4
ssh: connect to host 192.168.0.4 port 22: Connection refused
MacBook:~ sebastien$ ssh root@192.168.0.4 -p 2222
root@192.168.0.4's password: 
[root@(none) ~]# cd /mnt/onboard/.adds/koreader/
[root@(none) koreader]# ls
COPYING                  enable-wifi.sh           nickel.sh                settings
README.md                ffi                      obtain-ip.sh             settings.reader.lua
cache                    fonts                    ota                      settings.reader.lua.old
clipboard                frontend                 plugins                  setupkoenv.lua
common                   git-rev                  reader.lua               styletweaks
crash.log                history                  release-ip.sh            tar
data                     history.lua              resources                temp
datastorage.lua          jit                      restore-wifi-async.sh    tools
defaults.lua             koreader.sh              rocks                    zsync
defaults.persistent.lua  l10n                     screenshots
disable-wifi.sh          libs                     sdcv
dropbearmulti            luajit                   send2ebook
[root@(none) koreader]# cat settings.reader.lua
-- we can read Lua syntax here!
return {
    ["copt_font_gamma"] = 15,
    ["tap_to_follow_links"] = false,
    ["start_with"] = "last",
    ["disable_kobolight"] = true,
    ["frontlight_max_warmth_hour"] = 23,
    ["calibre_opds"] = {
        ["host"] = "192.168.0.2",
        ["port"] = "8080"
    },
    ["swipe_to_follow_nearest_link"] = true,
    ["statistics"] = {
        ["is_enabled"] = false,
        ["min_sec"] = 5,
        ["convert_to_db"] = true,
        ["max_sec"] = 120
    },
    ["hyph_right_hyphen_min"] = 1,
    ["opds_servers"] = {
        [1] = {
            ["title"] = "Project Gutenberg",
            ["url"] = "http://m.gutenberg.org/ebooks.opds/?format=opds"
        },
        [4] = {
            ["title"] = "Internet Archive",
            ["url"] = "http://bookserver.archive.org/catalog/"
        },
        [2] = {
            ["title"] = "Feedbooks",
            ["url"] = "http://www.feedbooks.com/publicdomain/catalog.atom"
        },
        [3] = {
            ["title"] = "ManyBooks",
            ["url"] = "http://manybooks.net/opds/index.php"
        }
    },
    ["home_dir"] = "/mnt/onboard/.hidden/library",
    ["cre_font"] = "Bookerly",
    ["screensaver_dir"] = "/mnt/onboard/.adds/screensaver",
    ["frontlight_intensity"] = 1,
    ["frontlight_warmth"] = 0,
    ["wikipedia_save_dir"] = "/mnt/onboard/.hidden/Wikipedia",
    ["SSH_port"] = 2222,
    ["copt_page_margins"] = {
        [1] = 20,
        [2] = 20,
        [3] = 20,
        [4] = 20
    },
    ["hyph_alg_default"] = "French.pattern",
    ["dicts_disabled"] = {},
    ["screensaver_delay"] = "3",
    ["language"] = "C",
    ["ignore_power_sleepcover"] = false,
    ["wikipedia_languages"] = {
        [2] = "en",
        [1] = "fr"
    },
    ["activate_menu"] = "tap",
    ["night_mode"] = false,
    ["show_file_in_bold"] = false,
    ["copt_status_line"] = 1,
    ["copt_css"] = "./data/epub_sbu.css",
    ["dict_largewindow"] = true,
    ["copt_embedded_css"] = 1,
    ["auto_restore_wifi"] = false,
    ["filemanagermenu_tab_index"] = 5,
    ["quickstart_shown_version"] = 201512585,
    ["show_hidden"] = true,
    ["swipe_to_go_back"] = true,
    ["home_dir_display_name"] = "~",
    ["screensaver_type"] = "random_image",
    ["lastfile"] = "/mnt/onboard/.hidden/library/1. BESTOF/Tolkien, J. R. R. - The Lord of the Rings 50th anniversary - eng.epub",
    ["natural_light_config"] = {
        ["red_offset"] = 0,
        ["white_offset"] = -50,
        ["red_gain"] = 24,
        ["green_offset"] = -65,
        ["white_gain"] = 30,
        ["exponent"] = 0.15,
        ["green_gain"] = 24
    },
    ["is_frontlight_on"] = false,
    ["reader_footer_mode"] = 1,
    ["copt_embedded_fonts"] = 0,
    ["wifi_was_on"] = false,
    ["device_id"] = "34BC6322D0534DE6922213D4A631A74C",
    ["coverbrowser_initial_default_setup_done"] = true,
    ["inbox_dir"] = "/mnt/onboard/.hidden/Wikipedia",
    ["hyph_left_hyphen_min"] = 2,
    ["footer"] = {
        ["disable_progress_bar"] = true,
        ["frontlight"] = true,
        ["disabled"] = false,
        ["book_time_to_read"] = false,
        ["toc_markers"] = false,
        ["wifi_status"] = false,
        ["battery"] = false,
        ["pages_left"] = true,
        ["all_at_once"] = true,
        ["page_progress"] = true,
        ["mem_usage"] = false,
        ["time"] = true,
        ["percentage"] = false,
        ["chapter_time_to_read"] = false
    },
    ["lastdir"] = "/mnt/onboard/.hidden/library",
    ["fallback_font"] = "Bookerly",
    ["floating_punctuation"] = 0,
    ["frontlight_auto_warmth"] = false,
    ["collate"] = "strcoll"
}
[root@(none) koreader]#

**Comment by KenMaltby** [comment]
I presume that when you have the calibre content server running (and the "Connect/share" balls are green) you have an option to: "Stop Content server [192.168.0.2, port 8080]" ?

**Comment by cramoisi** [comment]
@KenMaltby  : yes :/ (there is no ball on mac for the opds, only for the wireless btw)
But Ok, I'm in. It was a installed for testing and forgotten macos process which broke the thing.

**Comment by cramoisi** [comment]
yup I typo httpS by some sort of habitude :) my 8080 et 9090 (god, why !?)  were only accessible by the computers for a test I don't remember, so the fact I could access them from mine and my wife's and not with the reader...
I'm happy this is over, this thing bother me for months (and I won't even use it haha)
Thanks, people you are precious :)
